src/components/general/SideSheet/Modal/styles.less

```diff
@@ -0,0 +1,94 @@
+@import '~styles/mixins.less';
+@import '~styles/variables.less';
+@import '~styles/colors.less';
```

Comment:
No need to import variables or colors. They're available as global variables within the fusion root component. In fact, they clutter the transpiled code by adding `:root { --grid-unit: 8; // etc. }` a bunch of times to the output and the DOM